app/models/validations/shared_validations.rb

```diff
@@ -96,7 +98,8 @@ def inactive_status(date, resource)
            when :deactivated then open_deactivation.deactivation_date
            end
 
-    { scope: status, date: date&.to_formatted_s(:govuk_date), deactivation_date: closest_reactivation&.deactivation_date&.to_formatted_s(:govuk_date) }
+    scope = status
+    { scope:, date: date&.to_formatted_s(:govuk_date), deactivation_date: closest_reactivation&.deactivation_date&.to_formatted_s(:govuk_date) }
```

Comment:
What is the benefit of this?

Reply:
ah fair point this is from a previous approach, cheers